src/Atmos/Model/AtmosModel.jl

```diff
@@ -0,0 +1,122 @@
+
+import CLIMA.DGmethods: BalanceLaw, dimension, vars_aux, vars_state, num_state_for_gradtransform,  vars_gradtransform, vars_diffusive,
+  flux!, source!, wavespeed, boundarycondition!, gradtransform!, diffusive!,
+  init_aux!, init_state!, preodefun!
+
+struct AtmosModel{T,M,R} <: BalanceLaw
+  turbulence::T
+  moisture::M
+  radiation::R
+end
+
+using CLIMA.PlanetParameters: R_d, cp_d, grav, cv_d, MSLP, T_0
+
+const γ_exact = 7 // 5
```

Comment:
We shouldn't need \gamma anywhere (where it appears, some call to MoistThermo is in order)